README.md

```diff
@@ -29,6 +29,40 @@ jobs:
           head: master
 ```
 
+## Auto approve
+
+If you use a workflow which does not allow to merge pull requests without a review 
+("Require pull request reviews before merging" in your [repo settings](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuring-protected-branches))
+you can set `auto_approve` to `true`. In that case you'll have to provide a [personal access token](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token)
+for a user which is allowed to review the pull requests changes. Make sure the token has at least
+`public_repo` permissions and store the token inside of the [repository secrets](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository).
+
+An example workflow would then look like this:
+
+```yml
+name: Sync Fork
+
+on:
+  schedule:
+    - cron: '*/30 * * * *' # every 30 minutes
+  workflow_dispatch: # on button click
+
+jobs:
+  sync:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: tgymnich/fork-sync@v1.2
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          owner: llvm
+          base: master
+          head: master
+          auto_approve: true
+          personal_token: ${{ secrets.PERSONAL_TOKEN }}
```

Comment:
shouldn't the already used `GITHUB_TOKEN` have sufficient permissions to review and approve PRs?

Reply:
Tested it and but Github gave me an error message like "you cannot approve your own pull request" which makes sense somehow ;-)